src/quest_manager/models.py

```diff
@@ -975,3 +988,55 @@ def get_minutes_to_complete(self):
 
         minutes = (self.first_time_completed - self.timestamp).total_seconds() / 60
         return minutes
+
+
+class QuestionType(models.Model):
+    id = models.CharField(
```

Comment:
Don't override id, `id` is a default field all model.Models come with that is equivalent to a primary key (.pk).    `.id` is used all over the place and will be assumed it is a unique identifier for the object.

We don't need to generate primary keys for our models, the default id field works and is what everyone will expect.